python/device/device.py

```diff
@@ -310,7 +314,7 @@ def get_session_metrics(self):
             "runTime": str(self.metrics.run_time),
             "runState": str(self.metrics.run_state),
             "exitReason": self.metrics.exit_reason,
-            "pairingId": self.pairing_id,
+            Fields.Deprecated.PAIRING_ID: self.pairing_id,
```

Comment:
I'm going to move other strings into thief_constants.py, but not in this change.